src/test/java/cucumber/testSteps/CartPageSteps.java

```diff
@@ -0,0 +1,20 @@
+package cucumber.testSteps;
+
+import cucumber.TestContext;
+import io.cucumber.java.en.Then;
+import cucumber.pageObjects.CartPage;
+
+public class CartPageSteps {
+    private TestContext testContext;
+    private CartPage cartPage;
+
+    public CartPageSteps(TestContext context){
+        this.testContext = context;
+        this.cartPage = testContext.getPageObjectManager().getCartPage();
+    }
+    @Then("the product should be in the cart")
+    public void the_product_should_be_in_the_cart() {
```

Comment:
please do not mix conventions. In some places you are using camel case, here snake case.